helmfile.d/values/falco/falco-common.yaml.gotmpl

```diff
@@ -95,6 +95,32 @@ customRules:
   {{- end }}
   overwrites.yaml: |-
     {{- if .Values.falco.rulesFiles.default.enabled }}
+    # Dropping these rules since they are a relatively week indication of intrusion.
```

Comment:
```suggestion
    # Dropping these rules since they are a relatively weak indication of intrusion.
```

Reply:
Thank you for spotting that, I fixed it now.